test/benchmark/run_performance_tracker.sh

```diff
@@ -3,4 +3,19 @@
 # change to script directory
 cd "${0%/*}" || exit
 
+# verify if the benchmark/sift directory exists and if it doesn't create it
+if [ ! -d "./sift" ]; then
+  mkdir ./sift
+fi
+
+# check that the files sift_base.fvecs and sift_query.fvecs exist in the benchmark/sift directory.
+# download them otherwise.
+if [ ! -f "./sift/sift_base.fvecs" ] || [ ! -f "./sift/sift_query.fvecs" ]; then
+  echo "Downloading SIFT dataset"
+  wget ftp://ftp.irisa.fr/local/texmex/corpus/sift.tar.gz -P /tmp/
+  tar -xvf /tmp/sift.tar.gz -C /tmp
+  mv /tmp/sift/sift_base.fvecs  /tmp/sift/sift_query.fvecs ./sift/
+  rm -r /tmp/sift /tmp/sift.tar.gz
```

Comment:
Since these files are written to `/tmp` anyway, do we even need to `rm` them? Always a bit hesitant to have implicit `rm`'s in a script

Reply:
Not strictly needed, but I was thinking mostly on the scenario in which somebody would run the benchmark locally and the compressed files of hundreds of megas would be left in `/tmp`. I'll remove it.